### PR TITLE
support multiple contact IDs using strings instead of ints. 

### LINKED ISSUE #21: Support multiple contact groups
Current StatusCake configuration only supports one contact_id. Statuscake has updated their API to accept a list of contact IDs provided as a string, separating each contact ID with a comma. The library should update accordingly.

Documentation at: https://www.statuscake.com/api/Tests/Updating%20Inserting%20and%20Deleting%20Tests.md

### PR DESCRIPTION
**Pull request by tiagoboldt**
Closes #21.

### CODE REVIEW

**Comment by tiagoboldt** [comment]
Hi @croman, mind taking a peak at this? We want to make this features available with Terraform and they depend on this Statuscake client. Thanks

https://github.com/terraform-providers/terraform-provider-statuscake/pull/9

File: tests_test.go [hunk 1]
```diff
@@ -77,6 +77,7 @@ func TestTest_ToURLValues(t *testing.T) {
 		PingURL:        "http://example.com/ping",
 		Confirmation:   1,
 		CheckRate:      500,
+    ContactID:      "1,2",
```
Comment (croman) [inline tests_test.go]: The indentation here is incorrect, could you fix it?

**Comment by croman** [comment]
The tests are passing now, there's only one small indentation issue and then it's good to merge.

**Comment by tiagoboldt** [comment]
Hi @croman, latest commit fixed it. Spaces vs tabs, that eternal discussion.. Thanks :)

**Pull request merged**